.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
```

Comment:
this should be `true`

Reply:
Why?
Do you actually want people to ignore the templates altogether?

Reply:
Not all issue posts fit a template - it's not about ignoring the templates, it's about allowing for special cases. I've experienced myself how frustratingly awful issue templates are when your post doesn't fit a template when trying to post Paper issues